test-data/local/ECLIPSE/simple/SIMPLE.DATA

```diff
@@ -0,0 +1,59 @@
+RUNSPEC
+START
+1 JAN 2017 /
+
+DIMENS
+ 10 10 10 /
+OIL
+UNIFOUT
+WATER
+WELLDIMS
+ 0 /
+EQLDIMS
+/
+TABDIMS
+   1* 1* 30 30 /
+GRID
+
+SPECGRID
+ 10 10 10 1 F / 
+DX
+ 1000*1.0 /
+DY
+ 1000*1.0 /
+DZ
+ 1000*1.0 /
+TOPS
+ 100*1.0 /
+PORO
+ 1000*0.1 /
+PERMX
+ 1000*150 /
+PERMY
+ 1000*150 /
+PERMZ
+ 1000*15 /
+PROPS
+SWOF
+ 0.0 0.0  1.0 0.0
+ 1.0 1.0  0   0.0 /
+DENSITY
+ 3*40 /
+PVTW
+ 400    1    1.0E-06    1      0 /
+PVTO
+  0.001 15.0  1.0 0.1 /
+  1.6 5000.0  1.8 0.5
+      9000.0  1.7 0.6 /
+/
+SOLUTION
+EQUIL
+ 3*1000 0 1000 0 0 0 0 /
```

Comment:
High pressure at shallow depths :) 

If it does not matter for values in the test you can have 
`1000 100 1000 /` 
instead here to have something slightly more meaningful (item 5 is not relevant in this oil-water problem)

Reply:
So in total:
```
3*1000 0 1000 1000 100 1000 /
```
?

Reply:
I think I changed it to what you wanted now.